prime-angular-frontend/src/app/modules/adjudication/shared/components/dated-content-table/dated-content-table.component.scss

```diff
@@ -1,3 +1,15 @@
+@import 'palette';
+
 th {
   width: 1%;
 }
+
+.flex {
+  display: flex;
+  justify-content: space-between;
+}
+
+div.margin-right {
+  font-size: 13px;
```

Comment:
We probably want to stay away from defining font sizes in px and lean towards using relative font sizing.
``` font-size: smaller ```

Also with fonts when using an exact sizing you want to lean towards using em over px.